src/stories/Dropdown.stories.tsx

```diff
@@ -35,7 +35,7 @@ for (let i = 0; i < 100; i++) {
 LongList.args = {
   options: longListItems,
   label: "Dropdown Label",
-  labelPosition: "top",
+  labelPosition: "left",
```

Comment:
Any particular reason for this label position difference here?

Reply:
oops, sorry. I changed that to see how it would look, but I forgot to change it back. do we want to keep it to showcase different states or would you like to keep it all uniform?

Reply:
Could be nice to have another story in there showing a different layout state?